mesonbuild/backend/vs2010backend.py

```diff
@@ -870,6 +870,46 @@ def _prettyprint_vcxproj_xml(self, tree, ofname):
             of.write(doc.toprettyxml())
         replace_if_different(ofname, ofname_tmp)
 
+    def gen_vcxproj_filters(self, target, ofname):
+        # Generate pitchfork of filters based on directory structure.
+        root = ET.Element('Project', {'ToolsVersion': '4.0',
+                                      'xmlns': 'http://schemas.microsoft.com/developer/msbuild/2003'})
+        filter_folders = ET.SubElement(root, 'ItemGroup')
+        filter_items = ET.SubElement(root, 'ItemGroup')
+        mlog.debug(f'Generating vcxproj filters {target.name}.')
+        found_folders = []
+        for i in target.sources:
+            path = i.fname.replace('/', '\\')
+            dirname = os.path.dirname(path)
+            while dirname:
+                if dirname and dirname not in found_folders:
+                    found_folders.append(dirname)
+                dirname = os.path.dirname(dirname)
+                
+        for folder in found_folders:
+            filter_element = ET.SubElement(filter_folders, 'Filter', {'Include': folder})
+            uuid_element = ET.SubElement(filter_element, 'UniqueIdentifier')
+            uuid_element.text = '{' + str(uuid.uuid4()).upper() + '}'
+
+        (sources, headers, objects, languages) = self.split_sources(target.sources)
```

Comment:
```suggestion
        sources, headers, objects, languages = self.split_sources(target.sources)
```
The parens aren't necessary, and I think one of our linters will complain about them

Reply:
Interesting, I copied this line from another part of the code that had the braces, but I shall take your word for it!